cypress/e2e/recieve-notice/recieve-notice-checkin.cy.js

```diff
@@ -79,7 +79,11 @@ describe('Recieving notice: Checkout', () => {
       ServicePoints.createViaApi(testData.userServicePoint);
       testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
       Location.createViaApi(testData.defaultLocation);
-      cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanTypeId = res[0].id; });
```

Comment:
method cy.getLoanTypes affected 26 files.Please review again.Does need it to delete?